### DISCUSSION TITLE
transferProxyTokenEncryptionAesKey: transfer-proxy-token-encryption-aes-key → What value?

### DISCUSSION THREAD

**Discussion by jerzy-wachowiak** [eclipse-edc/Connector#3110] [Q&A]
AES is a symmetric encryption algorithm, used in EDC for securing EDR (?). If a CX network consists of several EDCs all of them have to use the same aes-key. (The aes key is set as EDC global in 0.3.4 in https://github.com/eclipse-tractusx/tractusx-edc/blob/568c5b7448f47935551524b129f5c83d8b170529/charts/tractusx-connector/values.yaml#L510 )

1. Is the above statement correct?
2. If yes, what is the value of aes-key used for data exchange on INT (Hotel Budapest)?

**Comment by stephanbcbauer** [comment]
@jerzy-wachowiak thanks for the question. Additionally, I would like to ask for the difference between `encryptionAesKey` and `transfer-proxy-token-encryption-aes-key` THX

**Comment by jimmarino** [comment]
I believe this question should be for the Tractus-X project, not the EDC.

**Comment by jerzy-wachowiak** [comment]
Jim, thank you for the hint! I transferred the discussion to https://github.com/eclipse-tractusx/tractusx-edc/discussions/426
